e2e/settings/index.spec.ts

```diff
@@ -275,39 +275,36 @@ test("it is possible to edit variables", async ({ page }) => {
   ).toHaveValue(updatedType);
 });
 
-test("it is possible to handle broadcasts", async ({ page }) => {
-  const { broadcast } = await createBroadcasts(namespace);
+test("it is possible to update broadcasts", async ({ page }) => {
+  const { broadcast: broadcasts } = await createBroadcasts(namespace);
+  expect(broadcasts, "test data has been created").toBeTruthy();
```

Comment:
I moved this up so the steps from setting up the test to visiting the page are logically separated.

Did I understand correctly that you wanted to check setting up the test data was successful?

I updated the text accordingly. The text `broadcast should be truthy` just repeats what is already in the code.

If we add additional test descriptions, we should try to explain our intention in more general, less technical terms. For example, explain why we are doing this, rather than what we are doing.